src/templates/partials/stunnel.hbs

```diff
@@ -1,32 +1,34 @@
-# {{output.header}}
-# {{{output.link}}}
+; {{output.header}}
+; {{{output.link}}}
 
-{{#if (minver "1.0.1" form.opensslVersion)}}
-    {{#if (minver "5.50" form.serverVersion)}}
-        {{#if (minver "1.1.0" form.opensslVersion)}}
-            {{#if (minver "1.1.1" form.opensslVersion)}}
+{{#if (minver "5.50" form.serverVersion)}}
+{{#unless (includes "TLSv1" output.protocols)}}
+{{#unless (includes "TLSv1.1" output.protocols)}}
+{{#if (includes "TLSv1.2" output.protocols)}}
+sslVersionMin = TLSv1.2
```

Comment:
We may need to explicitly set TLS1 as min version if present in `protocols` for 5.50+ though as its default may change anytime.

(Haven't checked the docs if they support it at all, what are the defaults etc. … but it may warrant a comment whether that version can be achieved at all if not etc.)

Reply:
Might consider `sslVersionMin = output.protocols[0]`
There are some *.hbs which use similar and assume the protocols list is sorted older to newer protocols, which is currently the case in src/static/guidelines/latest.json `tls_versions`.  TBD.

Reply:
also noticed the implicit reliance on array order: https://github.com/janbrasna/ssl-config-generator/issues/23